lib/groonga-query-log/response-comparer.rb

```diff
@@ -59,16 +59,55 @@ def same_response?
       @response1.body == @response2.body
     end
 
+    def same_sort_key?
+      return false unless @command.has_key?(:sort_keys)
```

Comment:
This is not good. It doesn't work with `sortby`.

`@command.sort_keys` returns `sort_keys` or `sortby` and returns `[]` for no `sort_keys`/`sortby` case.